### PR TITLE
Make it easy to test permissions when username/password enabled

### PR DESCRIPTION
**Pull request by benilovj**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -82,15 +82,15 @@ def self.find_or_create_from_cis2_oidc(userinfo)
   end
 
   def is_medical_secretary?
-    return false unless Flipper.enabled?(:cis2)
+    return email.include?("admin") unless Flipper.enabled?(:cis2)
```
Comment (misaka) [inline app/models/user.rb:85]: This is one of those times that I find `unless` kinda hard to read. Wish there was a `Flipper.disabled?`

**Review by misaka — APPROVED** [review]
🚀

File: app/models/user.rb (hunk 1 above)
Comment (misaka) [inline app/models/user.rb:85]: I feel like this was unlikely to be tested up until now. But now that we have a way to test, is it worth at least a feature test to show that admins can't record a vaccination?

Comment (benilovj) [reply app/models/user.rb:85]: Added in [5971b22](https://github.com/nhsuk/manage-vaccinations-in-schools/pull/2110/commits/5971b22c8e74b949b247d1ebcc31eb358a3b097a)

**Review by misaka — APPROVED** [review]
🚀

**Pull request merged**